.github/CODEOWNERS.md

```diff
@@ -0,0 +1 @@
+* @aperdomob @germandavid85 @luigisamurai @yesidbalvin @Scot3004 @santirogu
```

Comment:
This file should only have these names: `* @Valeryibarra @renardete @leonleo997 @holgiosalos`

Reply:
agree